src/test/resources/database/car/delete-all-cars.sql

```diff
@@ -0,0 +1,2 @@
+DELETE
+FROM cars;
```

Comment:
You have empty file remove-toyota-car-from-cars-table. sql i suggest adding there a query to delete only that car from db